### ISSUE TITLE
Incompatible with Dynamic FPS

### ISSUE THREAD

**Issue by LostLuma** [BumbleSoftware/Fastload#108]
### What went wrong?

Using Dynamic FPS and FastLoad together causes the world loading screen to stay open forever.
I've explained the reason [in the original report](https://github.com/juliand665/Dynamic-FPS/issues/129#issuecomment-1738090215), and will create a PR with a fix for this issue shortly.

### Reproduction steps

- Install Dynamic FPS and FastLoad
- Attempt to join a Minecraft world

### Visual demonstration

https://files.lostluma.net/UJ1pWH.png

### Crash Report & Log

unneeded

### Fastload Config

Default Config

### Environment

Dynamic FPS 3.2.0
FastLoad 3.4.0

**Comment by PlayfulPiano** [comment]
Hey so thank you for figuring out this issue, but this mod fix seemed to have caused a separate issue in my modpack i'm using.
Beforehand I would've gotten hung at the world generating screen, but after I was able to load worlds fine.
But, now if I try viewing the mod map atlas' full screen, it causes a crash (crash report below).
[crash-2023-10-14_09.17.25-client.txt](https://github.com/BumbleSoftware/Fastload/files/12905584/crash-2023-10-14_09.17.25-client.txt)


This issue doesn't happen if I don't have this mod & dynamicfps installed.

**Comment by justjustjustjust** [comment]
Yeeeep, came here exactly beacuse of this reason. Installed the fix and everything is PERFECT now. ty

**Comment by LostLuma** [comment]
Note that as of Dynamic FPS 3.4.2 if Fastload 3.4.0 or earlier is present we fix this on our side, but if this mod ever gets updated again the PR should be merged before the next release.

**Comment by KirboSoftware** [comment]
@LostLuma is this still the case to this day?

**Comment by LostLuma** [comment]
Yes. We ship the fix for Minecraft 1.20.4 and lower (since FastLoad is not available and not useful in 1.20.5+, and the background blur instead of dirt texture also means we no longer have this optimization).

**Comment by KirboSoftware** [comment]
Alright, I will take a look at #109 and implement it on the multiloader version 👍 since I think this affects all affect loaders that we support currently.
